packages/core-components/src/components/Progress/Progress.tsx

```diff
@@ -36,6 +36,6 @@ export function Progress(props: PropsWithChildren<LinearProgressProps>) {
   return isVisible ? (
     <LinearProgress {...props} data-testid="progress" />
   ) : (
-    <Box display="none" />
+    <Box display="none" data-testid="progress-placeholder" />
```

Comment:
Hmm, wonder if this should just be `"progress"` as well? It's kinda the behavior you'd expect I think, although it may break existing tests.

Reply:
Changed locally to "progress", some tests timed out but didn't seem like change was the cause. Let's see on CI.

Reply:
@Rugvip All tests passes except some UI tests that ran over the limit. Seems like we can go forward with this?

Reply:
@Rugvip Is there a chance we'll go forward with this?